create_fv3_mapping.f90

```diff
@@ -20,6 +20,10 @@ program create_fv3_mapping
   character*100      :: ims_lat_name = "imslat_4km_8bytes.bin"
   character*100      :: ims_lon_name = "imslon_4km_8bytes.bin"
 
+! SMAP input info
+  character*100      :: smap_path = "/scratch2/BMC/gsienkf/Zofia.Stanley/projects/fv3_mapping/"
```

Comment:
I'll move this file, and email you a new location.

Reply:
Done.